cmd/translate/main_integration_test.go

```diff
@@ -99,3 +99,8 @@ func mustGetFreePort() string {
 
 	return strconv.Itoa(addr.Port)
 }
+
+// ptr returns pointer to the passed in value.
+func ptr[T any](v T) *T {
+	return &v
+}
```

Comment:
this is used multiple places, please move to pkg to avoid redeclaring again and again

Reply:
Just a local helper func.
which pkg do you think this should belong to?

Reply:
`pkg/util` ? or something like that

Reply:
not local, it is used atleast 3 times in different places, and I forsee that it will be used more
<img width="533" alt="image" src="https://github.com/translate-agent/translate/assets/69954124/e36ec7bd-6e7b-44a4-98f3-dad6a69cf8c8">